CMakeLists.txt

```diff
@@ -87,8 +87,8 @@ file(GLOB xml app/conf/*.xml)
 file(GLOB scripts app/scripts/*.template)
 file(GLOB ipynb report/*.ipynb)
 file(GLOB sh app/scripts/*.sh report/*.sh)
-file(GLOB train_with_me_app app/conf/train-with-me/*)
-file(GLOB train_with_me_scripts app/scripts/train-with-me/*)
+file(GLOB train_with_me_conf app/conf/train-with-me/*)
```

Comment:
Then, you're not handling the new var below

Reply:
Done.